components/Card.js

```diff
@@ -0,0 +1,43 @@
+import { medications } from "@/lib/db";
+import styled from "styled-components";
+
+const CardContainer = styled.div`
+  border: 0px groove #ccc;
+  padding: 1.3em;
+  border-radius: 5px;
+  margin: 1.5em;
+  flex-direction: column;
+  justify-content: space-between;
+  box-shadow: 0 2px 8px rgba(0, 0, 0, 0.26);
+  @media (max-width: 375px) {
+    margin: 5%;
+  }
+`;
+const Title = styled.h1`
+  font-size: large;
+  margin-bottom: 10px;
+  text-shadow: 0 1px 2px rgba(0, 0, 0, 0.32);
+
+  @media (max-width: 375px) {
+    font-size: 1.2rem;
+    word-wrap: break-word;
+  }
+`;
+const Summary = styled.p`
+  font-size: medium;
+  margin-bottom: 10px;
+  text-shadow: 0 1px 2px rgba(0, 0, 0, 0.32);
+  word-wrap: break-word;
+`;
+export default function Card() {
+  return (
+    <div>
+      {medications.map((medication) => (
+        <CardContainer key={medication.id}>
+          <Title>{medication.name}</Title>
+          <Summary>{medication.summury}</Summary>
+        </CardContainer>
+      ))}
+    </div>
```

Comment:
I would love to see more semantic code. My suggestion:

```jsx
    <ul>
      {medications.map((medication) => (
        <li key={medication.id}>
          <CardContainer>
            <Title>{medication.name}</Title>
            <Summary>{medication.summury}</Summary>
          </CardContainer>
       </li>
      ))}
    </ul>
```

And: 
- your `CardContainer` could be an `article` instead of a `div`.
- `CardContainer` is a good name. You could also just go with `Card`.
- One key of your example data has a typo: it is **summary** and not **summury** I think you need to change this in your db.js too.